src/extensions/extension-manager.ts

```diff
@@ -8,9 +8,10 @@ import { extensionPackagesRoot } from "./extension-loader"
 import { getBundledExtensions } from "../common/utils/app-version"
 
 export interface InstalledExtension {
-  manifest: LensExtensionManifest;
-  manifestPath: string;
-  isBundled?: boolean; // defined in package.json
+  readonly manifest: LensExtensionManifest;
+  readonly manifestPath: string;
+  readonly isBundled?: boolean; // defined in package.json
```

Comment:
Even if you leave this as an interface, this should not be optional. Also, which `package.json`. I know it means `Lens`'s but it could mean an extension's. Please update the comment.